yt_dlp/extractor/vimeo.py

```diff
@@ -211,21 +212,15 @@ def _parse_config(self, config, video_id):
                     'width': int_or_none(key),
                     'url': thumb,
                 })
-            thumbnail = video_data.get('thumbnail')
-            if thumbnail:
-                thumbnails.append({
-                    'url': thumbnail,
-                })
+            thumbnails.extend(traverse_obj(video_data, (('thumbnail', 'thumbnail_url'), {'url': {url_or_none}})))
 
         owner = video_data.get('owner') or {}
-        video_uploader_url = owner.get('url')
 
         return {
             'id': str_or_none(video_data.get('id')) or video_id,
             'title': video_title,
             'uploader': owner.get('name'),
-            'uploader_id': video_uploader_url.split('/')[-1] if video_uploader_url else None,
-            'uploader_url': video_uploader_url,
+            'uploader_id': int_or_none(owner.get('id')),
```

Comment:
revert, this code still works for me
```suggestion
            'uploader_id': video_uploader_url.split('/')[-1] if video_uploader_url else None,
            'uploader_url': video_uploader_url,
```